boards/nucleo-f446ze/Makefile.features

```diff
@@ -8,6 +8,7 @@ FEATURES_PROVIDED += periph_can
 FEATURES_PROVIDED += periph_pwm
 FEATURES_PROVIDED += periph_rtc
 FEATURES_PROVIDED += periph_spi
+FEATURES_PROVIDED += periph_adc
```

Comment:
Kconfig needs an update too

Reply:
I'm not familiar with the Kconfig - but i hope one line from [commit](https://github.com/RIOT-OS/RIOT/pull/18647/commits/f13bc78fab055309dacaab540569b815864ae7fd) is fine.

Reply:
Yes that will work - please squash